src/components/ebay-checkbox/examples/WithLabel.marko

```diff
@@ -9,7 +9,7 @@ class {}
         on-change("emit", "change")
         on-focus("emit", "focus")
         on-keydown("emit", "keydown") />
-    <label class="field__label field__label--end" for="checkbox">
+    <label class=["field__label", "field__label--end", input.disabled && "field__label--disabled"] for="checkbox">
```

Comment:
I think what might be better is to actually have another story with disabled example, rather than using the toggle.
The toggles should only be used for arguments that get passed to the actual component.

Reply:
Should be moved now, thanks Andrew.